.env.sample.l3

```diff
@@ -0,0 +1,42 @@
+############################### DEFAULT #####################################
+# Chain ID
+CHAIN_ID=167006
+
+# Exposed ports
+PORT_L3_EXECTION_ENGINE_HTTP=8546
```

Comment:
will this conflict with the ws port for L2 execution engine? maybe we should do 8547 here and 8548 if so.